osbenchmark/builder/cluster_builder.py

```diff
@@ -0,0 +1,12 @@
+class ClusterBuilder:
```

Comment:
It would be nice to have some comments describing the roles and responsibility for each of these classes

Reply:
That's a good callout, will add comments